protocol/governance/cannonfile.satellite.toml

```diff
@@ -0,0 +1,63 @@
+name = "synthetix-governance-satellite"
+description = "On-chain voting for synthetix councils - Logic for satellite chains"
+version = "<%= package.version %>"
+include = ["./tomls/proxy-base.toml", "./tomls/council-token.toml"]
+
+[setting.salt]
+defaultValue = "governance-satellite"
+
+# Deployment Owner, defaults to first hardhat account
+[setting.owner]
+defaultValue = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
+
+[setting.initial_epoch_index]
+defaultValue = "0"
+
+[setting.initial_council_member]
+defaultValue = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
+
+[contract.AssociatedSystemsModule]
+artifact = "contracts/modules/core/AssociatedSystemsModule.sol:AssociatedSystemsModule"
+
+[contract.CrossChainModule]
+artifact = "contracts/modules/core/CrossChainModule.sol:CrossChainModule"
+
+[contract.ElectionModuleSatellite]
+artifact = "contracts/modules/core/ElectionModuleSatellite.sol:ElectionModuleSatellite"
+
+[contract.CouncilTokenModule]
+artifact = "contracts/modules/council-nft/CouncilTokenModule.sol:CouncilTokenModule"
+
+[router.CoreRouter]
+contracts = [
+    "AssociatedSystemsModule",
+    "CrossChainModule",
+    "ElectionModuleSatellite",
+    "InitialModuleBundle",
+]
+depends = [
+    "contract.AssociatedSystemsModule",
+    "contract.CrossChainModule",
+    "contract.ElectionModuleSatellite",
+    "contract.InitialModuleBundle",
+]
+
+[invoke.upgrade_core_proxy]
+target = ["InitialProxy"]
+from = "<%= settings.owner %>"
+func = "upgradeTo"
+args = ["<%= contracts.CoreRouter.address %>"]
+factory.CoreProxy.abiOf = ["CoreRouter"]
+factory.CoreProxy.event = "Upgraded"
+factory.CoreProxy.arg = 0
+depends = ["contract.InitialProxy", "router.CoreRouter"]
+
+[invoke.init_election_module_satellite]
```

Comment:
if you remove the `_satellite` postfix the step name could be overridden by the mothership cannonfile with its own init function